### PR TITLE
[DOCUMENTATION] First draft of working with emails

### PR DESCRIPTION
**Pull request by katebee**
## What does this change?

💌 💌

Ever wanted to understand EMAIL? Burning questions about forms and ExactTarget? 

Do you want MORE documentation in frontend?

## What is the value of this and can you measure success?

With this one simple guide, all your email related questions will be answered FOREVER!

🌈 🌈 

...Alternatively, at least you can Git Blame and know who to ask about stuff

## Does this affect other platforms - Amp, Apps, etc?

Nope

----------
Requesting feedback from: 
@gtrufitt 
@desbo 
@svillafe 
@guardian/dotcom-platform

### CODE REVIEW

File: docs/03-dev-howtos/18-working-with-emails.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+- [article](https://www.theguardian.com/email/form/article/37) >> Commonly used as embed in articles (example here). Button defaults to blue but tone overrides can be applied.
+- [footer](https://www.theguardian.com/email/form/footer/37) >> This is the one used in the footer on every page.
+- [plain](https://www.theguardian.com/email/form/plain/37) >> No header or description. Button will always be guardian-brand blue
+- [plaintone](https://www.theguardian.com/email/form/plaintone/3743) >> No header or description. Submit button splits from input on widescreen. Button defaults to blue but tone overrides can be applied. Used in [email interactives](https://www.theguardian.com/info/ng-interactive/2016/dec/07/sign-up-for-the-flyer).
+- [plaindark](https://www.theguardian.com/email/form/plaindark/3745) Used by [Documentaries](https://www.theguardian.com/world/ng-interactive/2016/oct/14/desert-fire-the-world-cup-rebels-of-kurdistan-video). Depreciated in favour of plaintone (which has a [dark option](http://localhost:9000/email/form/plaintone/3745)) and potentially removable, however, only if all usages can be found.
```
Comment (gtrufitt) [inline docs/03-dev-howtos/18-working-with-emails.md]: `deprecated` :)

Comment (katebee) [inline docs/03-dev-howtos/18-working-with-emails.md]: Cheers, probably, should have run this through a spell checker.. will do that now!

File: docs/03-dev-howtos/18-working-with-emails.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+These interactives have their [own repo](https://github.com/guardian/interactive-email-signups) and readme.
+
+
+## Writing a new email sign up form
+
+If you are adding a custom email sign up form somewhere, you will want to POST an email address and listID to https://www.theguardian.com/email.
+
+An example implementation of this can be found in  [**newsletters.scala.html**](https://github.com/guardian/frontend/blob/master/applications/app/views/signup/newsletters.scala.html#L74).
+
+### How to do email address validation
```
Comment (gtrufitt) [inline docs/03-dev-howtos/18-working-with-emails.md]: @svillafe take a 👀 😄

**Review by gtrufitt — APPROVED** [review]
Superb, nice one :)

**Comment by stephanfowler** [comment]
This is a great docs addition, nice one Kate.

**Pull request merged**